src/main/groovy/io/openliberty/tools/gradle/tasks/GenerateFeaturesTask.groovy

```diff
@@ -48,7 +49,21 @@ class GenerateFeaturesTask extends AbstractFeatureTask {
     private static final String BINARY_SCANNER_MAVEN_TYPE = "jar";
     private static final String BINARY_SCANNER_MAVEN_VERSION = "latest.release";
 
+    private static final String BINARY_SCANNER_CONFLICT_MESSAGE1 = "A working set of features could not be generated due to conflicts " +
+            "between configured features and the application's API usage: %s. Review and update your server configuration and " +
+            "application to ensure they are not using conflicting features and APIs from different levels of MicroProfile, " +
+            "Java EE, or Jakarta EE. Refer to the following set of suggested features for guidance: %s";
+    private static final String BINARY_SCANNER_CONFLICT_MESSAGE2 = "A working set of features could not be generated due to conflicts " +
+            "between configured features: %s. Review and update your server configuration to ensure it is not using conflicting " +
+            "features from different levels of MicroProfile, Java EE, or Jakarta EE. Refer to the following set of " +
+            "suggested features for guidance: %s";
+    private static final String BINARY_SCANNER_CONFLICT_MESSAGE3 = "A working set of features could not be generated due to conflicts " +
+            "in the application’s API usage: %s. Review and update your application to ensure it is not using conflicting APIs " +
+            "from different levels of MicroProfile, Java EE, or Jakarta EE.";
+    private static final String BINARY_SCANNER_CONFLICT_MESSAGE4 = "[None available]"; // format should match JVM Set.toString()
+
```

Comment:
Should we consider putting these strings in a ci.common util class since they are the same across ci.maven/ci.gradle? Can be a future enhancement

Reply:
Yes, I'm going to work on a binary scanner handler class.